### DISCUSSION TITLE
Broadcast to icecast2

### DISCUSSION THREAD

**Discussion by newlaurent62** [jamulussoftware/jamulus#1717] [General]
Hello, 

Just wanted to share what I'm working on as a user.

My group of musician uses Linux, Windows and MacOS.

In order to broadcast a musician mix made by a sound ingeneer, I would like to use : 
- jack, jacktrip and jamulus on musician side.
- jacktrip, Ardour and butt on sound ingeneer side
- icecast2 server for webradio
- vlc on phones, tablets and computers connected to icecast2 server.

Jamulus is used for musician sound. 
Jacktrip is used to send musician tracks to Ardour or Reaper. It can have a high latency in order to insure that transmitted sound is of better quality.
Butt write the mix stream on icecast2 server.

Kind Regards,

**Comment by hoffie** [comment]
Cool, thanks for letting us know!

Just for reference, there is also a PR in flight (#967) which will support feeding raw audio to external programs such as ffmpeg which should enable direct streaming to Icecast. However, when wanting to run further processing in the pipeline, something like your setup may still be the best option.
